optuna/visualization/matplotlib/_edf.py

```diff
@@ -23,7 +23,8 @@
 def plot_edf(study: Union[Study, Sequence[Study]]) -> "Axes":
     """Plot the objective value EDF (empirical distribution function) of a study with Matplotlib.
 
-    .. seealso::  optuna.visualization.plot_edf
+    .. seealso::
+        Please refer to :func:`optuna.visualization.plot_edf` for an example.
```

Comment:
Not a strong opinion, but how about adding a short explanation about the relationship between `optuna.visualization.plot_edf` and `optuna.visualization.matplotlib.plot_edf`? The following is just an example:
```suggestion
        Please refer to :func:`optuna.visualization.plot_edf` for an example, where this function can be replaced with it.
```